umake/frameworks/ide.py

```diff
@@ -53,7 +53,7 @@ def _add_to_group(user, group):
             output = subprocess.check_output(["adduser", user, group])
             logger.debug("Added {} to {}: {}".format(user, group, output))
             return True
-        except subprocess.CalledProcessError as e:
+        except subprocess.CalledProcessError:
```

Comment:
Yeah, add it to logger.error please (but same, in the other "refactoring" PR and in a separate commit).